packages/vant/src/image-preview/ImagePreviewItem.tsx

```diff
@@ -251,8 +252,12 @@ export default defineComponent({
     };
 
     const checkClose = (event: TouchEvent) => {
-      const isClickOverlay = event.target === swipeItem.value?.$el;
+      const swipeItemEl: HTMLElement = swipeItem.value?.$el;
+      const imageEl = swipeItemEl.firstElementChild;
```

Comment:
When using slots, it's also possible that the image element is not the first element, should we consider that case

Reply:
Fortunately, the parent element of the image slot is always fixed, so we don't have to consider this situation.

```js
<div class={bem('image-wrap')}>
  {slots.image({ src: props.src })}
</div>
```

Reply:
nice 😄